src/test/java/perkpack/controllers/PerkRestControllerTest.java

```diff
@@ -146,4 +150,35 @@ public void votePerkTest() throws Exception {
 
         accountRepository.delete(account);
     }
+
+    @Test
+    @WithMockUser(username = "five@gmail.com", password = "password")
+    public void changeScoreTest() throws Exception {
+        Category startingCategory = new Category("Another Test Category");
+        Perk startingPerk = new Perk("50% off", "Everything", startingCategory);
+        categoryRepository.save(startingCategory);
+        perkRepository.save(startingPerk);
+
+        Account user = new Account("Guy", "Five", "five@gmail.com", "password");
+        user = accountRepository.save(user);
+        PerkVote pv = new PerkVote("50% off", 1,user);
+
+        ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
+        String perkVoteJSON = ow.writeValueAsString(pv);
+
+        mockMvc.perform(post("/perks/vote").
+                contentType(jsonContentType).
+                content(perkVoteJSON)).
+                andExpect(status().isOk()).
+                andExpect(jsonPath("$.score", is(1)));
+        
+        mockMvc.perform(post("/perks/vote").
```

Comment:
I had to add this to your test because leaving the entries in the DB was causing other tests to fail.